Userland/Applications/ImageViewer/ViewWidget.h

```diff
@@ -34,8 +35,16 @@ class ViewWidget final : public GUI::Frame {
     int toolbar_height() { return m_toolbar_height; }
     bool scaled_for_first_image() { return m_scaled_for_first_image; }
     void set_scaled_for_first_image(bool val) { m_scaled_for_first_image = val; }
+    void set_path_for_first_image(const String& path)
+    {
+        m_path = path;
```

Comment:
Take this `path` as `String` and `move()` it into `m_path` instead.
```suggestion
    void set_path_for_first_image(String path)
    {
        m_path = move(path);
```